index.html

```diff
@@ -92,7 +92,7 @@
       </div>
       <!--/.item-->
       
-      <div class="item" style="background-image: url(images/slider/bg23.jpg)">
+      <div class="item" style="background-image: url(images/slider/bg2.jpg)">
         <div class="container">

```

Comment:
@LoneWolf47 the fix looks good but only issue is with the commit message, please change it